include/snowflake/client.h

```diff
@@ -287,6 +298,44 @@ typedef struct sf_snowflake_column_desc {
     sf_bool null_ok;
 } SF_COLUMN_DESC;
 
+/**
+ * Encryption material
+ */
+typedef struct sf_encryption_material {
+    char *query_stage_master_key;
+    char query_id[SF_UUID4_LEN];
+    int64 smk_id;
+} SF_ENC_MAT;
```

Comment:
Btw, do you need to expose SF_ENC_MAT, SF_STAGE_CRED, SF_STAGE_INFO to users? If not, please move them to client_int.h or other headers inside `lib`. The basic idea here is `client.h` should include only public constants and functions.